layouts/shortcodes/alert.html

```diff
@@ -1,12 +1,12 @@
-<div class="alert alert-{{ with .Get "context" }}{{.}}{{ else }}doks{{ end }} d-flex" role="alert">
-  <div class="flex-shrink-1 alert-icon">{{ with .Get "icon" }}{{.}} {{ end }}</div>
-  {{ with .Get "text"}}
-    <div class="w-100">{{ . | safeHTML }} </div>
-  {{ else }}
-    {{ with .Inner}}
-      <div class="w-100"> {{ . | markdownify}}</div>
-    {{ else }}
-      {{ errorf "No valid text variable or Inner content given"}}
-    {{ end }}
-  {{ end}}
+<div class="alert alert-{{-  with .Get "context"  -}}{{- . -}}{{-  else  -}}doks{{-  end  }} d-flex" role="alert">
+  <div class="flex-shrink-1 alert-icon">{{- with .Get "icon" -}}{{.}} {{-  end  -}}</div>
+  {{-  with .Get "text" -}}
+    <div class="w-100">{{- . | safeHTML  -}} </div>
```

Comment:
The rest of the shortcodes have one space between hyphens e.g. `{{- with .Inner -}}` and here we have
`{{-<space><space>with .Get "text"<space>-}}` or `{{-<space><space>with .Inner<space>-}}`. If this is intended or not leading to unexpected behavior, OK.